console-src/src/components/ClientList/ClientList.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+import { BodyText } from "../../components/Text";
+import { OAuthClient } from "../../types/oAuthClient";
+import colors from "../../theme/colors";
+import copy from "../../copy";
+
+type ClientListProps = {
+  clients: OAuthClient[];
+};
+
+type ClientItemProps = {
+  client: OAuthClient;
+};
+
+const ClientItemContainer = styled(Link)`
+  display: block;
+  padding: 1rem;
+  height: 2rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid ${colors.veryLightGrey};
+`;
+
+const ClientItem = (props: ClientItemProps) => {
+  const { client } = props;
+
+  return (
+    <ClientItemContainer to={`/clients/${props.client.id}`}>
+      <BodyText className="without-padding primary-black">
+        {client.name}
+      </BodyText>
+      <BodyText className="without-padding">{client.redirectUri}</BodyText>
+    </ClientItemContainer>
+  );
+};
+
+const ClientListContainer = styled.div`
+  border-radius: 0.25rem;
+  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.15);
+`;
+
+const ClientListHeader = styled.div`
+  border-top: 0.25rem solid ${colors.secondaryPurple};
+  border-top-left-radius: 0.25rem;
+  border-top-right-radius: 0.25rem;
+  padding: 1rem;
+  height: 30px;
```

Comment:
Can text in `ClientListHeader` be wrapped to the next line?

Reply:
I don't think it would happen currently as this is some static text, but I agree with you that it should be a bit more flexible, I'll tweak the styles to handle these overflowing text cases

Reply:
I have changed to `min-height` in this commit 053eccaebf699e3f8c2ef19359d4f8a831433e08 so that if text is overflowing it will wrap and the container will expand its height